linux_os/guide/system/software/integrity/software-integrity/aide/aide_build_database/rule.yml

```diff
@@ -62,6 +62,7 @@ references:
     nist: CM-6(a)
     nist-csf: DE.CM-1,DE.CM-7,PR.DS-1,PR.DS-6,PR.DS-8,PR.IP-1,PR.IP-3
     pcidss: Req-11.5
+    srg: SRG-OS-000363-GPOS-00150,SRG-OS-000445-GPOS-00199
```

Comment:
Actually, `SRG-OS-000363-GPOS-00150` is not assigned to [RHEL-08-010359](https://stigaview.com/products/rhel8/v1r9/RHEL-08-010359/) , I only see [RHEL-08-010360](https://stigaview.com/products/rhel8/v1r9/RHEL-08-010360/).

Could you remove it from this rule? 
And an extra step would be to remove it in rule for `package_aide_installed` too.

Reply:
Thank you @yuumasato  I haven't noticed that. Fixed now.